modules/ROOT/pages/docker-ldap-synchronizer.adoc

```diff
@@ -0,0 +1,96 @@
+= Deploy LDAP Synchronizer with Docker
+
+:description: How to install and use the LDAP Synchronizer docker  distribution.
+
+How to install and use the LDAP Synchronizer docker distribution.
+
+This guide assumes you have a working docker environment.
+
+== Configuration
+
+=== Environment Variables
+
+All general connection settings( `bonita.properties`, `ldap.properties`
+and `log.properties`) are configurable with environment properties.
+
+Specific configuration available on `mapper.properties` and `sync.properties` could configurable with a copy of files in directory link by volume `/opt/conf/user-config`.
+
+Those files could be copied directly from ldap-syncrhonizer docker image with command to directory link to volume `/opt/conf/user-config` with  :
+
+[source,bash,subs="verbatim,quotes"]
+----
+docker cp $(docker create --rm quay.io/bonitasoft/bonita-ldap-synchronizer:*version*):/conf/default/mapper.properties *directory_link_to_volume*
+docker cp $(docker create --rm quay.io/bonitasoft/bonita-ldap-synchronizer:*version*):/conf/default/sync.properties *directory_link_to_volume*
+----
```

Comment:
Maybe here we can describe the steps to override the e.g. `sync.properties` configuration file? 
i.e.
* copy the file from the base image using the command above `docker cp [....]`.
* edit the file
* start the docker image with the volume `docker run [...] -v pwd:/opt/conf/user-config [..]`

Reply:
ok